cli/azd/pkg/alpha/alpha_feature_manager.go

```diff
@@ -77,32 +79,32 @@ func (m *FeatureManager) IsEnabled(featureId FeatureId) bool {
 	// guard from using the alphaFeatureManager from multiple routines. Only the first one will create the cache.
 	m.withSync.Do(m.initConfigCache)
 
+	enabled := false
+
 	// For testing, and in CI, allow enabling alpha features via the environment.
-	envName := fmt.Sprintf("AZD_ALPHA_ENABLE_%s", strings.ToUpper(string(featureId)))
+	envName := fmt.Sprintf("AZD_ALPHA_ENABLE_%s", strings.ReplaceAll(strings.ToUpper(string(featureId)), ".", "_"))
 	if v, has := os.LookupEnv(envName); has {
 		if b, err := strconv.ParseBool(v); err == nil {
-			return b
+			enabled = b
 		} else {
 			log.Printf("could not parse %s as a bool when considering %s", v, envName)
 		}
+	} else if allOn := isEnabled(m.userConfigCache, AllId); allOn {
+		//check if all features is ON
+		enabled = true
```

Comment:
One thing that could be interesting to know about is if `azd` users are using the `All` flag. If we think that's valuable, we could consider also adding `all` to the `config.features` attribute if this is present.

Reply:
I think it is interesting.  The `All` flag is a way to turn azd into full experimental :) and receive the latest alpha feature always (no need to manual turning on).   Looking at most of folks turning this on will help us to probably offer one only switch for all alpha features, or if no-one uses it, maybe we can remove the `all` option (less to support)